packages/patternfly-4/react-table/src/components/Table/Body.js

```diff
@@ -32,13 +33,6 @@ class ContextBody extends React.Component {
     };
   };
 
-  parentsExpanded(parentId) {
-    const { rows } = this.props;
-    return rows[parentId].hasOwnProperty('parent')
-      ? this.parentsExpanded(rows[parentId].parent)
-      : rows[parentId].isOpen;
-  }
-
   mapCells = (row, rowKey) => {
     const { headerData } = this.props;
     let shiftKey = Boolean(headerData[0] && headerData[0].extraParams.onSelect);
```

Comment:
It would be good to see more comments, for example it looks like the shiftKey can take values of 0 | 1 | 2 here and then I am not sure how it is used

Reply:
I added the comments, refactored that part and fixed a small bug in the process

Reply:
I hope it is readable now